### PR TITLE
Fix python compatibility test

### PR DESCRIPTION
**Pull request by anandolee**
Fix python compatibility test when a new generated code public imports an old version(2.6.1 or older) generated code.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by anandolee** [comment]
Ping Josh. The failure test CONFIG=php5.6_mac is not related.

File: src/google/protobuf/compiler/python/python_generator.cc [hunk 1]
```diff
@@ -1380,8 +1380,14 @@ void Generator::FixOptionsForMessage(const Descriptor& descriptor) const {
 void Generator::CopyPublicDependenciesAliases(
     const string& copy_from, const FileDescriptor* file) const {
   for (int i = 0; i < file->public_dependency_count(); ++i) {
+    string module_name = ModuleName(file->public_dependency(i)->name());
     string module_alias = ModuleAlias(file->public_dependency(i)->name());
-    printer_->Print("$alias$ = $copy_from$.$alias$\n", "alias", module_alias,
+    printer_->Print("try:\n"
+                    "  $alias$ = $copy_from$.$alias$\n"
+                    "except AttributeError:\n"
+                    "  $alias$ = $copy_from$.$module$\n",
```
Comment (haberman) [inline src/google/protobuf/compiler/python/python_generator.cc]: Please add a comment about the scenario where this is required. New generated code importing old generated code?

Comment (anandolee) [inline src/google/protobuf/compiler/python/python_generator.cc]: Done. Thanks

**Comment by haberman** [comment]
LGTM.

**Pull request merged**